utils/paddedToAddress.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Remove padding from address
+ *
+ * @param address
+ * @returns
+ */
+export const paddedToAddress = (address: string) => {
+  if (address.slice(0, 2) === '0x') {
+    address = address.slice(2)
+  }
+
+  while (address.slice(0, 2) === '00') {
+    address = address.slice(2)
+  }
```

Comment:
For the case of:

`0x000000000000000000000000002419a44c90ff9dd06ce378115e0bf32442d4a5`

You can do a simple:

replace `0x000000000000000000000000` by `0x`

The current script will break if the address starts with 00 which can happen. (it didnt and i was wrong here: https://github.com/lukso-network/wallet.universalprofile.cloud/pull/251#discussion_r1477030036 )

In that case it is always 32bytes to 20bytes so you need to remove 12 bytes so 24 times 0

`0x000000000000000000000000065682c3c4300e75e1a67837105e2ae93e1ff575`

->

`0x065682c3c4300e75e1a67837105e2ae93e1ff575`

https://explorer.execution.mainnet.lukso.network/address/0x065682c3c4300e75e1a67837105e2ae93e1ff575

For some reasons the burnt pix starts with 0.....

Reply:
I've added additional `isAddress` check and tests so it will work with addresses starting with `0` too.

Reply:
Why not just a searchAndReplace? but ok this will work